setup.py

```diff
@@ -32,7 +32,7 @@
     'readlike==0.1.2',
     'requests>=2.6.0,<3',  # uses semantic versioning (after 2.6)
     'ReParser==1.4.3',
-    'protobuf>=3.1.0,<3.2.0',
+    'protobuf>=3.1.0',
```

Comment:
Can you update the upper bound here instead of removing it?

Reply:
Sure.  The latest version, 3.6.1, works so I'll do that...